src/renderer/root-frame.tsx

```diff
@@ -54,8 +54,17 @@ export class RootFrame extends React.Component {
     lensProtocolRouterRendererInjectable: LensProtocolRouterRenderer,
   ) {
     catalogEntityRegistry.init();
-    extensionLoader.loadOnClusterManagerRenderer();
+
+    try {
+      const loadingExtensions = extensionLoader.loadOnClusterManagerRenderer();
+      const loadingBundledExtensions = loadingExtensions.filter(e => e.isBundled).map(e => e.loaded);
```

Comment:
I made this change just so I could test what the UI looked like after waiting for all extensions. 
```suggestion
      const loadingBundledExtensions = loadingExtensions;
```
It still seems that the extensions are still loading when the UI comes up? Or maybe this is still expected?

https://user-images.githubusercontent.com/40840436/146953159-4c3ea9f0-8158-4df6-ba40-4b806d54c27c.mov

Reply:
That change doesn't work since the array has objects `{extension: extension, loaded: loadingPromise}` so you'd want to have 

`const allExtensions = loadingExtensions.map(e => e.loaded)`

Reply:
oops, missed the map

Reply:
I still get the same result, are my expectations wrong?

Reply:
Interesting. I'll test it. There are couple of things why this could happen. This approach waits only on the extensions' `onActivate()` method to finish. React rendering can happen after that. I did try with Lens Spaces extension bundled and it did wait for it enough to do rendering immediately. Which kind of the point of the change anyway since we want to have the login screen ready when the app opens.